src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.domain;
+
+import racingcar.enums.Movement;
+
+public class Car {
+    private final String name;
+    private int position = 0;
+
+    public Car(String name) {
+        this.name = name;
+    }
+
+    public void move(int moveCondition) {
+        if (moveCondition >= Movement.MIN_MOVE_CONDITION.getValue()) {
+            position++;
+        }
+    }
```

Comment:
```suggestion
    public void move(int moveCondition) {
        if (isMovable(moveCondition)) {
            position++;
        }
    }

    private static boolean isMovable(int moveCondition) {
        return moveCondition >= Movement.MIN_MOVE_CONDITION.getValue();
    }
```
메소드 분리를 하면 가독성이 더 좋아질 것 같습니다!

Reply:
오 ㅎㅎ 훨씬 깔끔하네요 감사합니다 😀